File: samples/snippets/visualbasic/System/string.split/basic.vb
Module basic
    Public Sub Basic1()
        '<snippet1>
        Dim s As String = "Today" & vbTab & "I'm going to school"
        Dim subs As String() = s.Split(" "c, Char.Parse(vbTab))

        For Each substring In subs
            Console.WriteLine("Substring: " & substring)
        Next

        ' This example produces the following output:
        '
        ' Substring: Today
        ' Substring: I 'm
        ' Substring: going
        ' Substring: to
        ' Substring: school
        '</snippet1>
    End Sub
End Module

File: snippets/csharp/System/String/Split/basic.cs
using System;

namespace Split
{
    class Basic
    {
        public static void Basic1()
        {
            //<snippet1>
            string s = "Today\tI'm going to school";
            string[] subs = s.Split(' ', '\t');

            foreach (var sub in subs)
            {
                Console.WriteLine($"Substring: {sub}");
            }

            // This example produces the following output:
            //
            // Substring: Today
            // Substring: I'm
            // Substring: going
            // Substring: to
            // Substring: school
            //</snippet1>
        }
    }
}
